diff --git a/includes/s_header.php b/includes/s_header.php
deleted file mode 100644
index bf2a046..0000000
--- a/includes/s_header.php
+++ /dev/null
@@ -1 +0,0 @@
-<html><head><title>Administrator Area</title><meta http-equiv="Content-Type" content="text/html; charset=utf-8"><link rel=stylesheet href=st.css></head><body leftmargin=8 topmargin=8 marginwidth=0 marginheight=0>
\ No newline at end of file
